pylint/extensions/broad_try_clause.py

```diff
@@ -58,13 +58,15 @@ def _count_statements(self, try_node: nodes.TryExcept | nodes.TryFinally) -> int
     def visit_tryexcept(self, node: nodes.TryExcept | nodes.TryFinally) -> None:
         try_clause_statements = self._count_statements(node)
         if try_clause_statements > self.linter.config.max_try_statements:
-            msg = f"try clause contains {try_clause_statements} statements, expected at most {self.linter.config.max_try_statements}"
+            msg = (
+                f"try clause contains {try_clause_statements} statements, expected at"
+                f" most {self.linter.config.max_try_statements}."
```

Comment:
```suggestion
                f" most {self.linter.config.max_try_statements}"
```
The trailing `.` is causing the CI failure.